bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -4,837 +4,561 @@
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f %unit%"/>
-	</channel-type>
-
-	<channel-type id="maxTempThisWeek" advanced="true">
-		<item-type>Number:Temperature</item-type>
-		<label>Max Temp This Week</label>
-		<description>Maximum Temperature this week</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f %unit%"/>
-	</channel-type>
-
-	<channel-type id="maxTempThisMonth" advanced="true">
-		<item-type>Number:Temperature</item-type>
-		<label>Max Temp This Month</label>
-		<description>Maximum Temperature this month</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f %unit%"/>
-	</channel-type>
-
-	<channel-type id="temperatureTrend" advanced="true">
-		<item-type>String</item-type>
-		<label>Temp Trend</label>
-		<description>Temperature Evolution Trend</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%s">
-			<options>
-				<option value="up">up</option>
-				<option value="stable">stable</option>
-				<option value="down">down</option>
-			</options>
-		</state>
-	</channel-type>
-
-	<channel-type id="setpointTemp">
-		<item-type>Number:Temperature</item-type>
-		<label>Setpoint</label>
-		<description>Thermostat temperature setpoint</description>
-		<category>Temperature</category>
-		<state pattern="%.1f %unit%" readOnly="false"/>
-	</channel-type>
-
-	<channel-type id="setpointMode">
-		<item-type>String</item-type>
-		<label>Setpoint Mode</label>
-		<description>Chosen setpoint_mode (program, away, hg, manual, off, max)</description>
-		<state readOnly="false">
-			<options>
-				<option value="program">Following a weekly schedule</option>
-				<option value="away">Applying the -away- temperature as defined by the user</option>
-				<option value="hg">Frost-guard</option>
-				<option value="manual">Applying a manually set temperature setpoint</option>
-				<option value="off">Currently off</option>
-				<option value="max">Heating continuously</option>
-			</options>
-		</state>
-	</channel-type>
-
-	<channel-type id="ThermRelayCmd" advanced="false">
+	<channel-type id="monitoring-status">
 		<item-type>Switch</item-type>
-		<label>Heating Status</label>
-		<description>Indicates whether the furnace is heating or not</description>
-		<state readOnly="true"/>
-	</channel-type>
-
-	<channel-type id="ThermOrientation" advanced="true">
-		<item-type>Number</item-type>
-		<label>Orientation</label>
-		<description>Physical orientation of the thermostat module</description>
-		<state readOnly="true" pattern="%d"/>
-	</channel-type>
-
-	<channel-type id="timeUtc" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Measurement Time</label>
-		<description>Timestamp when data was measured</description>
-		<category>Time</category>
-		<state readOnly="true"/>
+		<label>Monitoring</label>
+		<description>Monitoring state of the camera</description>
+		<category>Window</category>
```

Comment:
Looks like you added the "Window" category on the wrong channel type. I guess it was intended for the "window-open" channel type.